src/PrestaShopBundle/Resources/views/Admin/Sell/Customer/Blocks/View/sent_emails.html.twig

```diff
@@ -23,34 +23,38 @@
  * International Registered Trademark & Property of PrestaShop SA
  *#}
 
-{% if customerInformation.sentEmailsInformation is not empty %}
-  <div class="card">
-    <h3 class="card-header">
-      <i class="material-icons">mail_outline</i>
-      {{ 'Last emails'|trans({}, 'Admin.Orderscustomers.Feature') }}
-      <span class="badge badge-primary rounded">{{ customerInformation.productsInformation.boughtProductsInformation|length }}</span>
-    </h3>
-    <div class="card-body">
-      <table class="table">
-        <thead>
-        <tr>
-          <th>{{ 'Date'|trans({}, 'Admin.Global') }}</th>
-          <th>{{ 'Language'|trans({}, 'Admin.Global') }}</th>
-          <th>{{ 'Subject'|trans({}, 'Admin.Global') }}</th>
-          <th>{{ 'Template'|trans({}, 'Admin.Global') }}</th>
+<div class="card customer-sent-emails-card">
+  <h3 class="card-header">
+    <i class="material-icons">mail_outline</i>
+    {{ 'Last emails'|trans({}, 'Admin.Orderscustomers.Feature') }}
+    <span class="badge badge-primary rounded">{{ customerInformation.sentEmailsInformation|length }}</span>
+  </h3>
+  <div class="card-body">
+  {% if customerInformation.sentEmailsInformation is not empty %}
+    <table class="table">
+      <thead>
+      <tr>
+        <th>{{ 'Date'|trans({}, 'Admin.Global') }}</th>
+        <th>{{ 'Language'|trans({}, 'Admin.Global') }}</th>
+        <th>{{ 'Subject'|trans({}, 'Admin.Global') }}</th>
+        <th>{{ 'Template'|trans({}, 'Admin.Global') }}</th>
+      </tr>
+      </thead>
+      <tbody>
+      {% for sentEmail in customerInformation.sentEmailsInformation %}
+        <tr class="customer-sent-email">
+          <td class="customer-sent-email-date">{{ sentEmail.date }}</td>
+          <td class="customer-sent-email-language">{{ sentEmail.language }}</td>
+          <td class="customer-sent-email-subject">{{ sentEmail.subject }}</td>
+          <td class="customer-sent-email-template">{{ sentEmail.template }}</td>
```

Comment:
I believe these classes are brought back from more recent code aren't they? I guess there's no harm in backporting them here.

Reply:
This was introduced here https://github.com/PrestaShop/PrestaShop/pull/17965 by @matks (thanks)

Reply:
Yes, there is no issue importing them and it will make the markup easier to target for either browser tests or JS scripts